CONTRIBUTING.md

```diff
@@ -10,6 +10,38 @@ Any kind of help is welcome, even if you just discover an issue and don't have t
 ## Filing issues
 While there's no specific template for creating a new issue, please take the time to create a clear description so that it is easy to understand the problem.
 
+## Using the playground 
+> With the integrated playground support, you can easily extend the scrapers and also enjoy the IDE debug feature if needed.
+
+Once you prepare the playground environment you will be able to set scraping options, debug using the IDE and get the scraped transactions as generated csv file.
+ 
+### Prepare the playground environment
+You should run those steps only one time to prepare the playground environment. 
+1. run command `npm install` to fetch library dependencies.
+2. run command `npm run playground:prepare` to transpile the playground code into `.playground-lib`.
+
+### Run a scraper
+To run a scraper you should:
+1. Provide the scraper credentials by running `npm run playground:setup`.
+1. Set scraper options in file  `.playground-lib/scrape-options.js`. Make sure you are not modifying the original file under `playground` folder.
+2. run `npm run playground:scrape`
+
+> To keep your credentials safe, they will be stored in a separated folder under your os home directory as defined in file `playground/definitions.js`. The default value is set to `{homedir}/.bank-scrapers-playground`.
+
+> The generated report will be added under `.playground-lib/output` folder. This folder is being ignored by github and npm to prevent exposing your data. 
+
+> Before your push your changes to github, it is your responsibility to review changes and make sure that only desired information are being commited. 
+
+### Make changes to scrapers
+The playground is using the transpiled files created in `lib` folder. 
+
+You should run command `npm run prepublishOnly` everytime after changing the sources so the playground will grab your changes.
+
+### Debug your changes using the IDE
+To run the playground scripts within your IDE debugger, make sure you use file `.playground-lib/scrape.js` 
+
+If you want to set breakpoints, you should add them to the transpiled code under `lib` folders. Any breakpoints added to the `src` folder will be ignored by the IDE. 
```

Comment:
agreed, quoted from my previous reply:
> A prerequisite will be to make sure that we can add breakpoints somewhere when using babel-cli, otherwise we are missing the point of having playground support

Reply:
@esakal 👍